src/seamless_communication/models/inference/translator.py

```diff
@@ -205,5 +206,5 @@ def predict(
             return text_out.sentences[0], None, None
         else:
             units = unit_out.units[:, 1:][0].cpu().numpy().tolist()
-            wav_out, sr_out = self.synthesize_speech(units, tgt_lang, spkr)
-            return text_out.sentences[0], wav_out, sr_out
+            wav_out = self.vocoder(units, tgt_lang, spkr, dur_prediction=True)
+            return text_out.sentences[0], wav_out, self.sr
```

Comment:
I would prefer more descriptive names. What does `sr` mean?

Reply:
I'll change it to `sample_rate`